.github/workflows/node.js.yml

```diff
@@ -0,0 +1,109 @@
+name: Node.js CI
+
+on:
+  push:
+    branches:
+      - jp-gha-conversion
+      - staging
+      # - master
+  pull_request:
+    branches:
+      - jp-gha-conversion
```

Comment:
this can be removed already and won't affect any testing you are doing

Reply:
you are never opening up another pull request to merge code into the `jp-gha-conversion` branch